dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -67,7 +67,16 @@ public void readBadlyDistributedFileTest() throws Exception
         HPCCFile file = new HPCCFile(datasets[1], connString , hpccUser, hpccPass);
         file.setProjectList("");
 
-        List<HPCCRecord> records = readFile(file, connTO);
+        List<HPCCRecord> records = readFile(file, connTO, false);
+        assertEquals("Not all records loaded",expectedCounts[1], records.size());
+    }
+
+    @Test
+    public void readWithForcedTimeoutTest() throws Exception
+    {
+        HPCCFile file = new HPCCFile(datasets[0], connString , hpccUser, hpccPass);
+
+        List<HPCCRecord> records = readFile(file, connTO, true);
```

Comment:
As far as I can see, connTO will be null here unless it is overwritten via system property, and getFileAccessExpirySecs is then defaulted in dfsclient.hpccfile to 120 secs, which seems like an unnecessary wait for forced timeout test. Why don't we lower the connTO for this test to 1 or 2 secs?